src/components/VerticalResults.tsx

```diff
@@ -1,7 +1,7 @@
 import { CardComponent } from '../models/cardComponent';
 import { useAnswersState } from '@yext/answers-headless-react';
 import { VerticalResultsDisplay } from './VerticalResultsDisplay';
-import { Pagination, PaginationCssClasses } from './Pagination';
+import { PaginationCssClasses } from './Pagination';
```

Comment:
I don't think we should include the pagination classes in vertical results anymore because they won't be used

Reply:
Yes, since we should only be using `verticalResultsContainer` and `verticalResultsLoading` now